cookbooks/magnetic_stripes/magnetic_stripes.prm

```diff
@@ -0,0 +1,250 @@
+# A model of a mid-ocean ridge that shows how magnetic lineations
```

Comment:
Could this prm be simplified by referencing (i.e., "include") the previous mid-ocean ridge prm file and only specifying the modified or new sections?

Reply:
It probably could be, but since these are teaching cookbooks that someone should be able to follow without going through other cookbooks, I would prefer to not base it on other cookbooks.

Reply:
Makes sense, happy to keep the PRM file in its current state.